src/bors/handlers/trybuild.rs

```diff
@@ -232,17 +233,27 @@ fn get_pending_build(pr: PullRequestModel) -> Option<BuildModel> {
         .and_then(|b| (b.status == BuildStatus::Pending).then_some(b))
 }
 
-fn auto_merge_commit_message(pr: &PullRequest, reviewer: &str) -> String {
+fn auto_merge_commit_message(
+    pr: &PullRequest,
+    reviewer: &str,
+    jobs: Option<Vec<String>>,
+) -> String {
     let pr_number = pr.number;
-    format!(
+    let mut message = format!(
         r#"Auto merge of #{pr_number} - {pr_label}, r={reviewer}
 {pr_title}
 
 {pr_message}"#,
         pr_label = pr.head_label,
         pr_title = pr.title,
         pr_message = pr.message
-    )
+    );
+    if let Some(jobs) = jobs {
+        for job in jobs {
+            message.push_str(&format!("\nci-job: {}", job));
```

Comment:
```suggestion
            message.push_str(&format!("\ntry-job: {}", job));
```
We actually renamed the tag since I created the bors issue :sweat_smile: I modified it in the issue.